lesson-01-cw721/exercise-04/exercise.md

```diff
@@ -29,19 +29,19 @@ use cw721::{NftInfoResponse, TokensResponse};   // Imports response types for qu
 
 Most NFT projects _will_ import and use `cw721` for enforcing the correct types of query responses, but more often what you need should be imported from the token collection contract.
 
-Below are common use cases where you likely _won't_ need to import `cw721`.
+Common use cases where you likely _won't_ need to import `cw721` include:
 
-- Minting tokens (minting is not implemented in `cw721`, import the minting message from the token collection contract)
+- Minting tokens (as minting is not implemented in `cw721`, import the minting message from the token collection contract)
```

Comment:
This change seems less clear to me

Reply:
But we can leave it in if you feel strongly about it

Reply:
I wanted to make it clear that, due to the lack of implementation of minting in the specifications, we need to import the minting message. However, this is not a mandatory change, so I can remove it.

```suggestion
- Minting tokens (minting is not implemented in `cw721`, import the minting message from the token collection contract)
```